### DISCUSSION TITLE
Neogit Commands for the commit buffer become part of commit message

### DISCUSSION THREAD

**Discussion by queue-tip** [NeogitOrg/neogit#1270] [General]
Whenever I use the commit buffer it looks like this. Those commands end up in the commit message as shown below. Is there anything that can be done to prevent the commands from being inserted, or maybe inserting them commented out?
![neogit-commit-window](https://github.com/NeogitOrg/neogit/assets/23458550/6b87bdb6-ef79-49ae-8f35-ec4e6a5910c1)

![Screenshot from 2024-04-30 19-24-47](https://github.com/NeogitOrg/neogit/assets/23458550/2911aa99-7122-44d8-b85b-ef6cca8b8256)

**Comment by CKolkey** [comment]
Have you set `core.commentchar` to be the string "auto"? Those lines should be commented out... and not prefixed with the word "auto".

**Reply by queue-tip** [reply]
No, I haven't redefined commentchar. I'm using the [AstroCommunity lazy spec](https://github.com/AstroNvim/astrocommunity/blob/main/lua/astrocommunity/git/neogit/init.lua) from AstroNvim. Thanks for the pointer to what I should be looking for.

**Reply by CKolkey** [reply]
Oh, cool, didn't know that one. The config value would be a git config setting, not a vim thing. You can see it with `git config` :)

**Reply by queue-tip** [reply]
Oh I see. I thought you meant it was a Neogit setting. I've explicitly set it to "#" in my gitconfig and everything's working properly now. Thanks for the help!
